Yoast/javascript
# Add one transition word for Russian

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds "кроме того" to Russian transition words.
* [Yoast SEO Free] Improves transition words analysis for Russian.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Add a Russian sentence that contains the transition word "кроме того".
* Check that this sentence gets recognized as containing transition words by the transition words assessment.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes #
